create-litr.Rmd

```diff
@@ -426,6 +426,24 @@ setup <- function(package_dir) {
     chunk_start <- "^(\n```+[a-zA-Z0-9_]+\n)"
     idx_block <- stringr::str_which(x, chunk_start)
     original_code <- knitr::knit_code$get()
+    # We first get indices of skipped chunks in original_code list
+    skipped_chunks <- which(sapply(original_code, function(x){
+      echo_attr <- attr(x, "chunk_opts")$echo
+      include_attr <- attr(x, "chunk_opts")$include
+      if(!is.null(echo_attr) | !is.null(include_attr)){
+        if(!is.null(echo_attr)){
+          return(!echo_attr)
+        } 
+        if(!is.null(include_attr)){
+          return(!include_attr)
+        }
+      } else{
+        return(FALSE)
```

Comment:
If I'm not mistaken, a more succinct way to do this would be with the function `isTRUE()`, which maps NULL to FALSE.  So I think you could do this in one line:

```
return(isTRUE(attr(x, "chunk_opts")$echo) || isTRUE(attr(x, "chunk_opts")$include))
```
But I haven't actually checked this, so I could be wrong.

Reply:
Nice, this one-liner works if I switch it to `isFALSE`

Reply:
Great! Oh I got it backwards.